packages/webapp/src/utils/localCrypto.ts

```diff
@@ -56,49 +60,66 @@ const setPwdHash = (uid: string, pwd: string): boolean => {
 	return true
 }
 
+const getPwdHash = (uid: string): string => {
+	return window.localStorage.getItem(uid.concat(HASH_PWD_KEY))
+}
+
 const testPassword = (uid: string, passwd: string) => {
 	const currentPwdHash = getPwdHash(uid)
-	const edata = window.localStorage.getItem(uid.concat(VERIFY_TEXT_KEY))
-	if (!currentPwdHash || !edata) {
-		return false
-	}
-
-	const dataBytes = CryptoJS.AES.decrypt(edata, currentPwdHash)
-	const data = dataBytes.toString(CryptoJS.enc.Utf8)
 
-	if (data !== VERIFY_TEXT) {
+	const salt = getSalt(uid.concat(SALT_KEY))
+	if (!currentPwdHash || !salt) {
 		return false
 	}
-	return true
-}
+	const encryptedPasswd = bcrypt.hashSync(passwd, salt)
 
-const getPwdHash = (uid: string): string => {
-	return window.localStorage.getItem(uid.concat(HASH_PWD_KEY))
+	return encryptedPasswd === currentPwdHash
 }
 
-const getCurrentUser = (): string => {
+const getCurrentUserId = (): string => {
 	return window.localStorage.getItem(CURRENT_USER_KEY)
 }
 
-const setCurrentUser = (uid: string) => {
+const setCurrentUserId = (uid: string) => {
```

Comment:
I changed this function name because it conflicted with the a recoil function by the same name that was introduced to `LoginForm`